### ISSUE TITLE
URI which returns error. Should endup in fallbacksource. 

### ISSUE THREAD

**Issue by vijayalakshmiin** [code-and-co/react-native-image-offline#11]
I have an OfflineImage like this

 <OfflineImage
              resizeMode="cover"
              key={'https://upload.wikimedia.org/wikipedia/commons/3/36/Hopetoun_falls11111.jpg'}
              style={[styles.imgBackground]}
              fallbackSource={FALLBACK_SOURCE}
              source={{
                uri: 'https://upload.wikimedia.org/wikipedia/commons/3/36/Hopetoun_falls11111.jpg',
              }}
            />

Basically its a a invalid uri that I am trying to load. It should go to fallback source right. Could you please let me know if i am missing something here

**Comment by BHesseldieck** [comment]
@vijayalakshmiin do you still need help? 

We tested this scenario when building the package and it worked. If you need deeper assistance we need to get more information about the images und your setup.

Cheers,
Ben
